tests/schema/test_get_schema_title.py

```diff
@@ -0,0 +1,26 @@
+import dataclasses
+
+import winter.core
+from winter.schema.generation import get_schema_title
+
+
+@dataclasses.dataclass()
+class DTO:
+    id: int
+
+
+def test_get_schema_title():
+    @winter.core.component_method
+    def method(dto: DTO, second_dto: DTO):
```

Comment:
It should return the same type. Only different types should return different schemas

Reply:
So you have to declare DTO in different files